src/charm.py

```diff
@@ -569,5 +607,67 @@ def _get_numerology(sub_carrier_spacing: Frequency) -> int:
     raise ValueError(f"Unsupported sub-carrier spacing: {sub_carrier_spacing}")
 
 
+def _get_first_usable_subcarrier(
+    absolute_frequency_ssb: ARFCN, dl_absolute_frequency_point_a: ARFCN, numerology: int
+) -> int:
+    """Calculate first usable subcarrier.
+
+    In this implementation only FR1 is supported.
+
+    Args:
+        absolute_frequency_ssb (ARFCN): Frequency-domain position of the SSB
+        dl_absolute_frequency_point_a (ARFCN): Frequency-domain position of Point A in Downlink
+        numerology (int): Numerology
+    """
```

Comment:
Can you document the return value? I'd like to know the units for example. I assume right now that there are none and that this is a kind of index, but it would be nice if it was documented here.